go/runtime/transaction/transaction.go

```diff
@@ -68,7 +77,8 @@ func (ak *artifactKind) UnmarshalBinary(data []byte) error {
 
 var (
 	// txnKeyFmt is the key format used for transaction artifacts.
-	txnKeyFmt = keyformat.New('T', &hash.Hash{}, artifactKind(0))
+	// TODO Matevz: What exactly does third argument represent? I wanted to use kindInvalid (as default?), but I'm getting crashes in transaction_test.go then.
```

Comment:
You can check the `keyformat` docs. The first argument is the type/tag while all following arguments specify sizes of key atoms. Only the type matters, not the value, so `artifactKind(0)` is correct. This is NOT a default value.

Reply:
Marshallable `artifactKind` is required in order to obtain its size in bytes. I changed this to first valid value now, i.e. `artifactKind(1)`.